hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionImplementation.java

```diff
@@ -161,7 +165,11 @@
   private final boolean hostnamesCanChange;
   private final long pause;
   private final long pauseForCQTBE;// pause for CallQueueTooBigException, if specified
-  private boolean useMetaReplicas;
```

Comment:
Let me explain a bit here. The old behavior for 'useMetaReplicas' is kept, no change of that. The flag is consolidated to the mode as the mode flag also supports the old behavior. Looking at the code, 'useMetaReplicas' only supports the region locator at this moment. The config is kept in case 'META_REPLICA_USE' needs to support a broader scope. This is just an implementation consolidation.

Reply:
OK, I got your point. Here the' useMetaReplicas' flag is only for locating region, so it is fine to remove it. The "hbase.meta.replicas.use" config will still be honored here, and at other places, we will still check the "hbase.meta.replicas.use" config separately.